### DISCUSSION TITLE
Welcome to vector-vscode-vcast Discussions!

### DISCUSSION THREAD

**Discussion by johnpaliotta** [vectorgrp/vector-vscode-vcast#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sesame0707** [comment]
Hello!

Short introduction:
I'm a junior software engineer working on a commercial project. I have really minor experience using VectorCAST - I have never created a single unit test using VectorCAST; I've only adjusted (using the VectorCAST GUI) a couple of them. At our project, we use VectorCAST 2022.sp11.

Purpose of my participation:
I'd like to replace/minimize the usage of the VectorCAST GUI by this VS Code extension. Hence, I'll post some questions in the "Q&A" discussion category about the problems that I face.

Best regards,
Piotr

**Reply by aytey** [reply]
Hi @sesame0707,

Feel free to raise any Q&A questions you might have about the extension, and we'd be delighted to answer them.

You are also free to reach-out to [support@vector.com](mailto:support@vector.com) and then we can assign a dedicated support person to help you with any VectorCAST queries you might have.

Happy testing -- cheers,

Andrew

**Reply by sesame0707** [reply]
Thank you very much in advance!

**Comment by wNightCats** [comment]
Hello，

I want to use AI to automatically generate some test cases for me, but I see that I need a tool called Reqs2X. Where can I download this tool?

Best regards,
Yibin

**Reply by Zbigor** [reply]
Hello Yibin,

thanks for reaching out. I suggest you check out https://www.vector.com/gb/en/products/products-a-z/software/vectorcast/ai-enhanced-vectorcast/ first and fill in the questionnaire.

Cheers,
Igor
